### ISSUE TITLE
Graphical glitch, garbled interface

### ISSUE THREAD

**Issue by ThibaultLemaire** [Microsoft/vscode#29637]
- VSCode Version: Code 1.13.1 (379d2efb5539b09112c793d3d9a413017d736f89, 2017-06-14T18:18:03.540Z)
- OS Version: Linux x64 3.16.0-4-amd64 Debian 3.16.43-2
  - Gnome Version : 3.14.1
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|vscode-markdownlint|DavidAnson|0.9.1|
|npm-intellisense|christian-kohler|1.3.0|
|path-intellisense|christian-kohler|1.4.2|
|vscode-eslint|dbaeumer|1.2.11|
|python|donjayamanne|0.6.5|
|gitlens|eamodio|4.2.0|
|vscode-npm-script|eg2|0.2.0|
|search-node-modules|jasonnutter|1.1.1|
|java|redhat|0.6.0|
|vim|vscodevim|0.9.0|
|nodejs-extension-pack|waderyan|0.1.9|
|JavaScriptSnippets|xabikos|1.4.1|;

---

Steps to Reproduce:

1. Launch VSCode
2. Switch to full-screen
3. Launch Node.js debugger

---

![screenshot from 2017-06-27 14-40-21](https://user-images.githubusercontent.com/21345269/27597872-f95b8730-5b63-11e7-80c3-eecdb46acacf.png)

Screenshot is unaltered except for the terminal area which has been blurred for privacy.
The garbled areas vanished one by one, when I hovered the cursor above them.

This happened randomly and I was unable to reproduce.

I am not familiar with VSCode's code, but I think it might be caused by Electron itself.

**Comment by mjbvz** [comment]
Can you please check if this reproduces when you start code with the `--disable-gpu` command line flag

**Comment by Naughtylus** [comment]
Well, as I've said, this happened only once, and I really don't know how. AFAIK, it could have been totally random and unrelated to Node.js debugger.

I've just tried the `--disable-gpu` mode, and nothing happened. I don't use code that often, but I'll make sure to try every time I get the chance. I'll post back if it reproduces.

**Comment by ThibaultLemaire** [comment]
It happened again! Okay, so it was in normal mode, I have yet to reproduce with `--disable-gpu`. I still don't know what triggers it exactly, I was in full-screen, but had no debugger running this time (should I remove the Node.js debugger from my initial description?).

I think it happened when I switched desktops back to code ; I've tried to quickly switch desktops, but all I could achieve was to crash the gnome shell... (Should I file that as well in an other issue?)
